Streetcode/Streetcode.WebApi/Utils/WebParsingUtils.cs

```diff
@@ -126,92 +127,105 @@ public async Task ParseZipFileFromWeb(string mode)
 
         // This method processes initial csv file and creates data.csv, where all duplicated addresses are removed in order to optimize the script.
         // This script also calls OSM Nominatim API in order to get lat and lon of current address row
-        public async Task ProcessCSVfile(string extractTo, bool deleteFile = false, params int[] columnsToExtract)
+        public async Task ProcessCSVfile(string extractTo, bool deleteFile = false)
```

Comment:
**Please respect naming conventions**
- No Async suffix, even though the method is asynchronous;
- CSV -> Csv, file -> File

Reply:
done